### DISCUSSION TITLE
Can't print entity labels

### DISCUSSION THREAD

**Discussion by HaychEmDee** [explosion/spaCy#12983] [Help: Coding & Implementations] [answered]
Lately, I've been trying to refresh my memory on natural language processing, and I wanted to practice named entity recognition. Problem is, the code I made can't print the entity labels for a sample sentence. What am I missing?

![entity label issue](https://github.com/explosion/spaCy/assets/133580949/a7a91a31-be55-47b3-94e7-925006b93c43)

**Comment by adrianeboyd** [comment, marked as answer]
There aren't any named entities, which are typically proper nouns, in your example sentence.

The spacy course is a good place to start for a refresher: https://course.spacy.io
